package.json

```diff
@@ -187,6 +187,11 @@
                     "type": "boolean",
                     "default": true,
                     "markdownDescription": "Enables code linting for HTML+HUBL and CSS+HUBL languages.\n\n*Requires a valid `hubspot.config.yml` file*"
+                },
+                "hubspot.allowUsageTracking": {
+                    "type": "boolean",
+                    "default": true,
+                    "markdownDescription": "Allow usage tracking within extension."
```

Comment:
Could we tie this option to the setting in the `hubspot.config.yml`? Or maybe just rely on config + VS Code setting? I think having 3 separate ways to opt in/out of tracking will get confusing.

Reply:
Agreed, even having the two between `hubspot.config.yml` and VSCode is a little confusing, but unavoidable if we want to use cli-lib functionality. I think we can stick to the two and remove this

Reply:
One thing that I'm worries about is if someone is using the extension capabilities outside of the config and wants to disable tracking 🤔